src/enum.lisp

```diff
@@ -135,12 +224,12 @@
          (%foreign-enum-value ,type ,value :errorp t)
          ,value)))
 
-;;; There are two exansions necessary for an enum: first, the enum
+;;; There are two expansions necessary for an enum: first, the enum
 ;;; keyword needs to be translated to an int, and then the int needs
 ;;; to be made indirect.
 (defmethod expand-to-foreign-dyn-indirect (value var body (type foreign-enum))
```

Comment:
good for you! i don't even know what they are... :)

i was missing a couple of words of documentation about these defgeneric based macro expand protocols. e.g. what is the intended difference between ```-translate``` and ```-convert```, etc.

now i think i have a partial picture, but it was very opaque when i first approached the codebase.